ksqldb-common/src/main/java/io/confluent/ksql/serde/SerdeOptions.java

```diff
@@ -60,17 +60,12 @@ private SerdeOptions(final ImmutableSet<SerdeOption> options) {
     validate(this.options);
   }
 
-  @SuppressWarnings("MethodMayBeStatic")
   public EnabledSerdeFeatures keyFeatures() {
-    // Currently there are no key features:
-    return EnabledSerdeFeatures.of();
+    return features(true);
```

Comment:
Ignore the changes in this file - this is work I need to do, but was needed to get the examples working.

Reply:
see:  https://github.com/confluentinc/ksql/pull/6256